src/cirrus/docs/components/workflows/definitions.rst

```diff
@@ -147,6 +147,49 @@ workflows. Cirrus allows several means of building such multi-stage pipelines:
   conditions don't quite fit with the in-the-box approaches.
 
 
+Error handling
+--------------
+
```

Comment:
Random thought - another thing that is confusing is the pre- and post-batch steps surrounding batch tasks in a workflow. Somewhere, we should explain why they exist, but also that a batch task may be marked as successful but fail in the post-batch step.

Caveat: I'm not sure if this is because the projects I've been on were "doing error handling wrong", often it was BYOC with a wide variety of logging and error handling strategies that we were trying to black box.

Reply:
https://cirrus-geo.github.io/cirrus-geo/stable/cirrus/components/workflows/batch.html#pre-batch-and-post-batch ?